src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -497,6 +497,45 @@ def get_custom_registry_credentials(cmd,
     )
 
 
+def build_timers_info(cmd, schedules):
+    TimerTrigger, TriggerStatus = cmd.get_models(
+        'TimerTrigger', 'TriggerStatus')
+    timer_triggers = []
+
+    # Provide a default name for the timer if no name was provided.
+    for index, schedule in enumerate(schedules, start=1):
+        # space?
+        split_schedule = None
+        if ':' in schedule:
+            split_schedule = schedule.split(":")
+        timer_triggers.append(
+            TimerTrigger(
+                name=split_schedule[0] if split_schedule else "defaultTimer" + str(index),
```

Comment:
Do you need to call strip() to remove the whitespace?

Reply:
See fix